### PR TITLE
161 apiv0 move documentation of finished api to openapi

### PR DESCRIPTION
**Pull request by 0815Sailsman**
Setup scalar as openapi client and write some docs to describe the v0 endpoints. [#161]

https://161-apiv0-move-documentation-of-finished-api-to-openapi.corewar.shonk.software/api/v0/docs

### CODE REVIEW

**Comment by 0815Sailsman** [comment]
btw, I won't fix the branch errors regarding scalar.js

these just pop up because we serve it manually ourselvers. If we were to install this as a node_module, those would not appear. And I don't think fixing the code smells of an external library is part of our scope...

**Review by TR1NITROTOLUOL — APPROVED** [review]
I think it is clear now how to move the documentation to openapi. It looks good to me.

File: backend/api/src/main/resources/openapi/v0/openapi.yaml
[the remark this answers is not in the document]
Comment (0815Sailsman) [reply backend/api/src/main/resources/openapi/v0/openapi.yaml:1]: I used intellij to generate the docs for the backend, and removed everything related to v1 endpoints.

**Review by nikkischnelle — APPROVED** [review]
lgtm

**Pull request merged**